hack/mirror/mirror_image_list.txt

```diff
@@ -0,0 +1,5 @@
+quay.io/prometheus/node-exporter  sealio/mirrored-node-exporter v1.6.0
+quay.io/prometheus-operator/prometheus-config-reloader  sealio/mirrored-prometheus-config-reloader v0.67.0
+quay.io/prometheus/prometheus  sealio/mirrored-prometheus v2.46.0
+registry.k8s.io/kube-state-metrics/kube-state-metrics  sealio/mirrored-kube-state-metrics v2.9.2
+quay.io/kubecost1/kubecost-cost-model  sealio/mirrored-kubecost-cost-model 1.105.2
```

Comment:
placing this file under the `hack` dir is strange, can we create a sub dir under `deploy` dir, named `images`, and put this file inside?